### DISCUSSION TITLE
BE启动失败，日志显示“All disks are broken, exit”

### DISCUSSION THREAD

**Discussion by chenerbo** [apache/doris#35516] [A - General / Q&A]
start time: Tue 28 May 2024 04:04:32 PM CST
INFO: java_cmd /opt/java/jdk1.8.0_311/bin/java
INFO: jdk_version 8
WARNING: Logging before InitGoogleLogging() is written to STDERR
W0528 16:04:32.409950 42314 doris_main.cpp:363] ignore broken disk, path = /data1/doris/data
F0528 16:04:32.409983 42314 doris_main.cpp:383] All disks are broken, exit.
*** Check failure stack trace: ***
*** Query id: 0-0 ***
*** tablet id: 0 ***
*** Aborted at 1716883472 (unix time) try "date -d @1716883472" if you are using GNU date ***
*** Current BE git commitID: ae923f7 ***
*** SIGABRT unknown detail explain (@0x3f00000a54a) received by PID 42314 (TID 42314 OR 0x7fd9a2040e00) from PID 42314; stack trace: ***
 0# doris::signal::(anonymous namespace)::FailureSignalHandler(int, siginfo_t*, void*) at /root/src/doris-2.0/be/src/common/signal_handler.h:417
 1# 0x00007FD9A2200090 in /lib/x86_64-linux-gnu/libc.so.6
 2# raise in /lib/x86_64-linux-gnu/libc.so.6
 3# abort in /lib/x86_64-linux-gnu/libc.so.6
 4# 0x0000564E2FC4AAC9 in /opt/doris/be/lib/doris_be
 5# google::LogMessage::SendToLog() in /opt/doris/be/lib/doris_be
 6# google::LogMessage::Flush() in /opt/doris/be/lib/doris_be
 7# google::LogMessageFatal::~LogMessageFatal() in /opt/doris/be/lib/doris_be
 8# main at /root/src/doris-2.0/be/src/service/doris_main.cpp:296
 9# __libc_start_main in /lib/x86_64-linux-gnu/libc.so.6
10# _start in /opt/doris/be/lib/doris_be

**Comment by ixzc** [comment]
中文问题，建议去ask.selectdb.com去问问题
